### DISCUSSION TITLE
Sentinel-2 cloud masking

### DISCUSSION THREAD

**Discussion by jessjaco** [microsoft/PlanetaryComputer#186] [General]
Has anyone applied cloud masking to sentinel-2 using something besides the scene classification band? (I haven't had the best luck with it in the past). I'm aware of a newer masking methodology (code [here](https://github.com/sentinel-hub/sentinel2-cloud-detector)) which can be applied on the Planetary Computer using Earth Engine data as described [here](https://towardsdatascience.com/creating-sentinel-2-truly-cloudless-mosaics-with-microsoft-planetary-computer-7392a2c0d96c). I wonder if there are any other approaches, using that algorithm or otherwise?

**Comment by TomAugspurger** [comment]
Just a quick note: s2-cloudless isn't an option because it operates on L1-C data, but the Planetary Computer provides L2-A data.

I've heard good things about the performance of https://github.com/azavea/cloud-buster, but I haven't tried applying it to data hosted on the Planetary Computer.
